lib/paper_trail/serializers/json.rb

```diff
@@ -41,6 +41,14 @@ def where_object_changes_condition(*)
           discussion at https://github.com/paper-trail-gem/paper_trail/issues/803
         STR
       end
+
+      # Raises an exception as this operation is not allowed from text columns.
+      def where_object_changes_from_condition(*)
+        raise <<-STR.squish.freeze
+          where_object_changes_from does not support reading JSON from a text
+          column. The json and jsonb datatypes are supported.
+        STR
+      end
```

Comment:
If we do explicitly raise in the [query class](https://github.com/paper-trail-gem/paper_trail/pull/1286/files#r553058607) itself, we can remove this method from each serializer, unless there's any other location I'm not aware of where this might get called via meta-programming in any other corners of the library I'm not familiar with.